src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  box-sizing: border-box;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars {
+  display: flex;
+  /* Don`t past a tests but looks correct */
+  /* gap: 4px; */
```

Comment:
The thing with flex-gap is that not all of the browsers support it yet, that's why tests doesn't take it to account